src/hub/interfaces/IHub.sol

```diff
@@ -15,7 +15,6 @@ interface IHub is IHubBase, IAccessManaged {
   /// @dev decimals The number of decimals of the underlying asset.
   /// @dev addedShares The total shares added across all spokes.
   /// @dev swept The outstanding liquidity which has been invested by the reinvestment controller, expressed in asset units.
-  /// @dev realizedPremiumRay The interest-free premium already accrued across all spokes, expressed in asset units and scaled by RAY.
   /// @dev premiumOffsetRay The total premium offset across all spokes, used to calculate the premium, expressed in asset units and scaled by RAY.
```

Comment:
Add in comment that value is a signed int now (and in other struct comments where value is used)